__snapshots__/src/index.test.js.md

```diff
@@ -171,110 +175,110 @@ Generated by [AVA](https://ava.li).
           <Tree
             data={
               Map {
-                '0' => {
+                'rddts-0' => {
                   _children: [
-                    '0-0',
-                    '0-1',
+                    'rddts-0-0',
+                    'rddts-0-1',
                   ],
                   _depth: 0,
-                  _id: '0',
+                  _id: 'rddts-0',
                   children: undefined,
                   label: 'item1',
                   value: 'value1',
                 },
-                '0-0' => {
+                'rddts-0-0' => {
                   _children: [
-                    '0-0-0',
-                    '0-0-1',
+                    'rddts-0-0-0',
+                    'rddts-0-0-1',
                   ],
                   _depth: 1,
-                  _id: '0-0',
-                  _parent: '0',
+                  _id: 'rddts-0-0',
+                  _parent: 'rddts-0',
                   children: undefined,
                   label: 'item1-1',
                   value: 'value1-1',
                 },
-                '0-0-0' => {
+                'rddts-0-0-0' => {
                   _depth: 2,
-                  _id: '0-0-0',
-                  _parent: '0-0',
+                  _id: 'rddts-0-0-0',
```

Comment:
The nodes are labeled as `[parentId-]depth[-arrayIndex]`. I didn't have a strong reason for doing so at the time but it felt like an easy enough thing to do and gives us some kind of information encoding for each node that doesn't have a preset id. It's borderline silly I guess but I'm not sure if anyone is relying on this pattern or not. The `-N` suffix throws off this scheme. We can simply have the id as `rdtsN` or `rdts.N` or something other than the `-` and the existing scheme would still be intact. Thoughts?

Reply:
Sure, I can add a separator prop when constructing the generator which we use blank or dot for.